### PR TITLE
Ropes Issue 5 - Adding tagging to images

### PR DESCRIPTION
**Pull request by Hiw**

### CODE REVIEW

**Comment by Hiw** [comment]
[Issue 5: Image Tags](https://trello.com/c/Ych4lJzW/365-issue-5-image-tags)

File: app/views/images/index.html.erb [hunk 1]
```diff
@@ -1,5 +1,6 @@
 <% @all_images.each do |image| %>
   <img src='<%= image.url %>'/>
+  <%= image.tag_list %>
```
Comment (eric199wen) [inline app/views/images/index.html.erb:3]: We should add tests for this, also for `show`, they should be in controller test.

Comment (Hiw) [reply app/views/images/index.html.erb]: Added tests

**Review by eric199wen — APPROVED** [review]
LGTM!

**Pull request merged**